### ISSUE TITLE
[sidebars] move away from icon font in favor of inline SVG

### ISSUE THREAD

**Issue by justvanrossum** [googlefonts/fontra#592]
Sidebars should become plug-ins. One step towards making that happen is using inline SVG for the icons, instead of using the custom icons font.

Note: we need inline SVG because we need to control the icon color cia CSS, as it depends on the light/dark theme.

Idea: create a web component called `inline-svg` that takes a `src` attribute. This could be used like this in HTML:

```html
<inline-svg src="/icons/point.svg"></inline-svg>
```

The web component would use the `fetch()` api to retrieve the file specified by `src`.

~@fabiocaccamo, what do you think?~

We agreed to do this.

Tasks:

- [x] Make icon export script, UFO -> SVG #593
- [x] Put exported SVG data in `/src/fontra/client/images/` #593
- [ ] Write `<inline-svg>` web component
- [ ] Convert sidebar icons to use `<inline-svg>`
- [ ] Convert toolbar icons to use `<inline-svg>`
- [ ] Convert text align icons to use `<inline-svg>`
- [ ] Delete fontra-icons.woff2 + CSS usage + build script

**Comment by fabiocaccamo** [comment]
@justvanrossum I like the idea, using/changing icons in the frontend must be as simple as possible.

**Comment by justvanrossum** [comment]
I will write a script that converts the glyphs in the icon UFO to SVG. I think it's best to use fonttools' `SVGPathPen` for this (which only makes the SVG path), and construct the basic SVG skeleton in code, so we can get it exactly how we want it.

**Comment by fabiocaccamo** [comment]
Will the script be used only once or everytime we need to convert a new icon to svg?

**Comment by justvanrossum** [comment]
For now we need to convert the existing icons from UFO to SVG, but I'm thinking that for Fontra's own icon needs it may be useful to maintain them in the UFO. We just export SVG instead of TTF.

**Comment by justvanrossum** [comment]
I've posted #593. We can either merge this first, and then continue the work in a new PR, or we work in that PR branch `svg-icons` to have a single PR for the issue.

**Comment by fabiocaccamo** [comment]
I think keeping issues and PRs separated is better in this case.
